internal/server/v1beta1/user_test.go

```diff
@@ -31,35 +31,74 @@ func TestGetUserStarredAssets(t *testing.T) {
 		size     = 10
 	)
 	type testCase struct {
+		UserUUID     string
```

Comment:
Adding this field to the test seems unnecessary.

Reply:
we need to pass empty uuid and valid uuid in different test cases. this field is required to store that.

Reply:
How does passing empty or non empty `UserUUID` affect any test? We are controlling the error code for validate user call.